figure_generation/manuscript_figure_generation.py

```diff
@@ -0,0 +1,566 @@
+panel_one.moveto(30,30)
+
+
+# In[4]:
+
+
+panel_two = (
+    sg.fromfile(f"{str(corpora_comparison_path)}/svg_files/biorxiv_pmc_frequency_odds.svg")
+)
+
+panel_two_size = (
+    np.round(float(panel_two.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_two.root.attrib['height'][:-2])*1.33, 0),
+)
+scale_x = 1.2
+scale_y = 1.2
+
+print(f"original: {panel_two_size}")
+print(f"scaled:{(panel_two_size[0]*scale_x, panel_two_size[1]*scale_y)}")
+
+panel_two = panel_two.getroot()
+panel_two.scale_xy(x=scale_x, y=scale_y)
+panel_two.moveto(30, 400)
+
+
+# In[5]:
+
+
+panel_three = (
+    sg.fromfile(f"{str(corpora_comparison_path)}/svg_files/biorxiv_pmc_frequency_bar.svg")
+)
+
+panel_three_size = (
+    np.round(float(panel_three.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_three.root.attrib['height'][:-2])*1.33, 0),
+)
+scale_x = 1.2
+scale_y = 1.2
+
+print(f"original: {panel_three_size}")
+print(f"scaled:{(panel_three_size[0]*scale_x, panel_three_size[1]*scale_y)}")
+
+panel_three = panel_three.getroot()
+panel_three.scale_xy(x=scale_x, y=scale_y)
+panel_three.moveto(630, 380)
+
+
+# In[6]:
+
+
+panel_four = (
+    sg.fromfile(f"{str(corpora_comparison_path)}/svg_files/preprint_published_frequency_odds.svg")
+)
+
+panel_four_size = (
+    np.round(float(panel_four.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_four.root.attrib['height'][:-2])*1.33, 0),
+)
+scale_x = 1.15
+scale_y = 1.15
+
+print(f"original: {panel_four_size}")
+print(f"scaled:{(panel_four_size[0]*scale_x, panel_four_size[1]*scale_y)}")
+
+panel_four = panel_four.getroot()
+panel_four.scale_xy(x=scale_x, y=scale_y)
+panel_four.moveto(30, 820, scale=1)
+
+
+# In[7]:
+
+
+panel_five = (
+    sg.fromfile(f"{str(corpora_comparison_path)}/svg_files/preprint_published_frequency_bar.svg")
+
+)
+
+panel_five_size = (
+    np.round(float(panel_five.root.attrib['width'][:-2])*1.33, 0), 
+    np.round(float(panel_five.root.attrib['height'][:-2])*1.33, 0),
+)
+scale_x = 1.15
+scale_y = 1.15
+
+print(f"original: {panel_five_size}")
+print(f"scaled:{(panel_five_size[0]*scale_x, panel_five_size[1]*scale_y)}")
+
+panel_five = panel_five.getroot()
+panel_five.scale_xy(x=scale_x, y=scale_y)
+panel_five.moveto(620, 800, scale=1)
+
+
+# In[8]:
+
+
+panel_one_label = sg.TextElement(10, 20, "A", size=22, weight="bold")
+panel_two_label = sg.TextElement(10, 400, "B", size=22, weight="bold")
+panel_three_label = sg.TextElement(610, 400, "C", size=22, weight="bold")
+panel_four_label = sg.TextElement(30, 810, "D", size=22, weight="bold")
+panel_five_label = sg.TextElement(620, 810, "E", size=22, weight="bold")
+
+
+# In[9]:
+
+
+figure_one = sg.SVGFigure("1400", "1200")
+figure_one.append([
+    panel_one, 
+    panel_two, 
+    panel_three,
+    panel_four,
+    panel_five,
+    panel_one_label,
+    panel_two_label,
+    panel_three_label,
+    panel_four_label,
+    panel_five_label
+])
+display(SVG(figure_one.to_str()))
```

Comment:
I think the figure will look more balanced if you expand the top plot/key to fill both columns. I'm not super familiar with SVGutils, so I don't know whether that is possible though

Reply:
(Fig. 1)

Reply:
I guess at a minimum, just to be nitpicky, i would align to the left even if you don't want to expand it so that it fits in the first column.

Also it looks like they all use the same `xyscale` but it looks like D and E are slightly smaller than B and C for some reason